src/audio/google/google_rtc_audio_processing.c

```diff
@@ -757,6 +777,7 @@ static struct module_interface google_rtc_audio_processing_interface = {
 	.prepare = google_rtc_audio_processing_prepare,
 	.set_configuration = google_rtc_audio_processing_set_config,
 	.get_configuration = google_rtc_audio_processing_get_config,
+	.trigger = trigger_handler,
```

Comment:
Generally C code styles prefer shorter, non-namespaced names for static symbols.  It's true this one file hasn't done that historically, but I guess I'd like to see it evolve in that direction.  (Not least because "google_rtc_audio_processing_" is a LOT of typing!)